docs/spec_draft.md

```diff
@@ -1410,6 +1411,39 @@ More formally, `result[i0, ..., iR-1]` is equal to:
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.popcnt
+
+### Semantics
+
+Counts the number of bits set in `operand` tensor element-wise and produces a
+`result` tensor.
+
```

Comment:
Performs element-wise count of the number of bits set in the `operand` and produces a `result` tensor.

or

Counts the number of bits set in `operand` tensor, element-wise, and produces a
`result` tensor.